OpenRA.Mods.Common/Pathfinder/BasePathSearch.cs

```diff
@@ -96,7 +96,7 @@ protected BasePathSearch(IGraph<CellInfo> graph)
 
 			// Determine the minimum possible cost for moving horizontally between cells based on terrain speeds.
 			// The minimum possible cost diagonally is then Sqrt(2) times more costly.
-			cellCost = graph.Actor.Trait<Mobile>().Locomotor.Info.TerrainSpeeds.Values.Min(ti => ti.Cost);
+			cellCost = graph.Actor.Info.TraitInfo<MobileInfo>().LocomotorInfo.TerrainSpeeds.Values.Min(ti => ti.Cost);
```

Comment:
Why not `((Mobile)graph.Actor.OccupiesSpace).Info.LocomotorInfo`?

Reply:
I tried, but couldn't get it to compile. Looks like I was inserting the `(Mobile)` at the wrong spot, though (before `Actor`, but after `graph`).